distributed/scheduler.py

```diff
@@ -20,7 +20,7 @@
     from cytoolz import frequencies, merge, pluck, merge_sorted, first
 except ImportError:
     from toolz import frequencies, merge, pluck, merge_sorted, first
-from toolz import valmap, first, second, compose, groupby
+from toolz import valmap, second, compose, groupby
```

Comment:
Is there a reason to prefer these come from `toolz` only instead of optionally coming from `cytoolz` as well?

Reply:
There is not reason I can think of. I wasn't aware of cytoolz until now, but it seems that from the ReadME it would be worth implementing, and see if it breaks anything:
https://github.com/pytoolz/cytoolz#cytoolz

Reply:
No worries. Went ahead and switched to optionally using `cytoolz` where available. ( https://github.com/dask/distributed/pull/3579 )

We've done that for similar functions in other places. So the risk of breaking is low.